internal/pkg/manifest/env.go

```diff
@@ -217,7 +217,16 @@ func (cfg *environmentHTTPConfig) loadLBConfig(env *config.CustomizeEnv) {
 	if env.IsEmpty() {
 		return
 	}
-	cfg.Public.Certificates = env.ImportCertARNs
+	if len(env.InternalALBSubnets) != 0 {
+		cfg.Private.Subnets = env.InternalALBSubnets
+	}
+	if env.ImportVPC != nil {
+		if len(env.ImportVPC.PublicSubnetIDs) == 0 {
+			cfg.Private.Certificates = env.ImportCertARNs
+			return
+		}
+		cfg.Public.Certificates = env.ImportCertARNs
```

Comment:
i think this is not backwards-compatible with the current version - this requires importing a VPC to be able to use imported certs.

i think the same should go for the private lb - let them use imported certs without importing a VPC.

Reply:
Fixed and added unit test.